app/src/main/java/org/p2p/wallet/common/date/ZonedDateExtensions.kt

```diff
@@ -51,5 +52,14 @@ fun Long.toZonedDateTime(): ZonedDateTime = ZonedDateTime.ofInstant(
     ZoneId.systemDefault()
 )
 
+fun String.toZonedDateTime(): ZonedDateTime {
```

Comment:
I am not sure, that this function should work this way. In case if string is not instance of ISO we should return null.
You can add extra function like  - `asZonedDateTimeOrNow()`